app/controllers/shipit/api/stacks_controller.rb

```diff
@@ -25,6 +25,11 @@ def show
         render_resource stack
       end
 
+      def destroy
+        stack.schedule_for_destroy!
+        render_resources stacks
```

Comment:
I don't think returning the list of stacks is the right API return value for a DELETE. Based on https://tools.ietf.org/html/rfc7231#section-4.3.5 it should probably be a 202 Accepted since the deletion hasn't happened yet, or maybe 204 if we know it will always work.